src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -3614,6 +3620,9 @@ Give the compiler some way to differentiate the methods. For example, you can gi
   <data name="ERR_BadSpecialByRefLocal" xml:space="preserve">
     <value>Parameters or locals of type '{0}' cannot be declared in async methods or lambda expressions.</value>
   </data>
+  <data name="ERR_BadSpecialByRefIterator" xml:space="preserve">
+    <value>foreach statement cannot operate on variables of type '{0}' in async or iterator methods.</value>
```

Comment:
Perhaps `A foreach statement ...` to avoid deciding whether to capitalize the keyword.